### PR TITLE
Migrate to go modules

### PR DESCRIPTION
**Pull request by ivan-p-nikolov**
# What 

"Modernize" the neoism fork
* Update  FT fork to use go modules.
* Move to major version 2

# Why

Snyk started reporting vulnerabilities in the `jmcvetta/neoism`. In order to address them we need to move the affected services to our fork and update the dependencies.

### CODE REVIEW

File: Dockerfile.tests [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM golang:1
+
+ENV PROJECT=concepts-rw-neo4j
```
Comment (karsov) [inline Dockerfile.tests]: `concepts-rw-neo4j` ?

File: README.md [hunk 2]
```diff
@@ -23,37 +19,26 @@ YMMV; use in production at your own risk.
 
 # Requirements
 
-[Go 1.1](http://golang.org/doc/go1.1) or later is required.
+[Go 1.13](http://golang.org/doc/go1.13) or later is required.
 
 Tested against Neo4j 2.2.4 and Go 1.4.1.
```
Comment (karsov) [inline README.md]: This line can also be updated.

File: Dockerfile.tests (hunk 1 above)
Comment (ivan-p-nikolov) [reply Dockerfile.tests]: Well stack overflow stared blocking my copy-paste. I needed to get the code from somewhere ...

File: README.md (hunk 2 above)
Comment (ivan-p-nikolov) [reply README.md]: My logic for not updating it was: "I need 1.13 to build with modules, but the code itself is tested with 1.4.1".
Now reading it again the "go" part is kind of useless. We only need `Tested against Neo4j 2.2.4`

File: README.md [hunk 3]
```diff
@@ -23,37 +19,25 @@ YMMV; use in production at your own risk.
 
 # Requirements
 
-[Go 1.1](http://golang.org/doc/go1.1) or later is required.
+[Go 1.13](http://golang.org/doc/go1.13) or later is required.
 
-Tested against Neo4j 2.2.4 and Go 1.4.1.
+Tested against Neo4j 3.4.0.
```
Comment (karsov) [inline README.md:24]: **Nitpicking:**

```suggestion
Tested against Neo4j 3.4.*.
```

**Review by karsov — APPROVED** [review]

**Review by mchompalova — APPROVED** [review]

**Pull request merged**